### ISSUE TITLE
review

### ISSUE THREAD

**Issue by Wang-tiantian** [Wang-tiantian/test#29]
**- 构图美学** 
  - step0
    - siglip 多模态（图文）对齐模型
    - 图像编码器（SiglipVisionModel）：一般是vit
    - 文本编码器（SiglipTextModel）：一般是transformers的encoder
    - 公共投影空间： 两侧编码器输出经过线性层投影后进行对齐（clip/对比损失）
    - comp_new_checkpoint_v2 两个头
    - slglip的visionmodel加自定义class头 输出2 和composition头 输出1    
  - step1 class
    - 在comp_new_checkpoint_v2的基础上 loss用bce 训练集为2分类0/1 输出为is crop和logits 两个头
    - datasets：输入文件夹路径list和路径和图像transform函数（resize totensor norm） 返回pixel_values和lables 2
    - dataloader：collectfc 按照bs返回 data和target
    - freeze head 且 freeze vit 为false 也就是全训
    - 不同层用不同学习率 头和vit
    - model输出logits和is crop
    - 把最小val loss的保存为best
  - step2 comp
    - 在step1的基础上 训练集为构图分数 loss用的l1 输出为logits和scores
    - datasets：输入文件夹路径list和路径和图像transform函数（resize totensor norm） 返回pixel_values和lable 1
    - dataloader：collectfc 按照bs返回 data和target
    - freeze head 为false 全训
    - model输出logits和comp value
    - 把最小val loss（class和comp的loss都有）的保存为best
**-  颜色编辑**
- 预先：分类模型 验证同一style
- 阶段一 styleid hidream ice dreamo 问题 不能很好的保留原图像 
- 阶段二 颜色编辑  
  - 先用florence给出框的坐标（人 ） 
  - 输给sam2的image predict分割（对人得mask取反得到背景）得到人与背景分开的照片 
  - 再给sam2自动分割  没全覆盖1_sam2   补全 test m2m可以不开 fill会合并很多东西
  - 或者给kmeans 按照颜色分割 自动全覆盖
  - 计算参数 应用 生成结果 判别 颜色奇怪还是因为分割 特别边缘问题
- 阶段三 fluxkontext微调
  - 快速尝试 24gb 5h a100
  - 微调优化
  - demo qwen指令优化 gradio
**- 姿态生成**
**- 预研：图像分割** 
- 洞察分析，洞察任务排序：
- 1. 运行模型进行能力复现（尽可能多跑一些非官方的图的效果）。
- 2. 精读相关的论文以及技术报告，做技术的基础分析，将该类技术的方法及路线都做一下整理。
- 3. 给出一些自己的观点（如：模型间的优缺点对比，该类方案使用成熟度等
- 图像编辑：图像分割 可控生成 姿态生成 风格迁移 
- 支撑任务：批跑 inpating 的 demo
